### PR TITLE
refactor tabular modification

### PR DESCRIPTION
**Pull request by dbraquart**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/dto/TabularModificationInfos.java [hunk 1]
```diff
@@ -35,11 +37,15 @@
...
 @ModificationErrorTypeName("TABULAR_MODIFICATION_ERROR")
 public class TabularModificationInfos extends ModificationInfos {
-    @Schema(description = "Modification type")
+
+    public static final String TABULAR_EQUIPMENT_TYPE = "tabularEquipmentType";
+
+    @Schema(description = "equipment type")
     @NonNull
-    private String modificationType;
+    private IdentifiableType equipmentType;
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/modification/server/dto/TabularModificationInfos.java:45]: I think it's not enough. We can have here types like GENERATOR_MODIFICATION, GENERATOR_CREATION or EQUIPMENT_DELETION

Comment (etiennehomer) [reply src/main/java/org/gridsuite/modification/server/dto/TabularModificationInfos.java:45]: Using ModificationType enum would be more appropriate

Comment (SlimaneAmar) [reply src/main/java/org/gridsuite/modification/server/dto/TabularModificationInfos.java:45]: No `ModificationType` represente all modification types of  network
We need only injection modificatio/creation/deletion

Comment (SlimaneAmar) [reply src/main/java/org/gridsuite/modification/server/dto/TabularModificationInfos.java:45]: operationType { MODIFICATION, CREATION, DELETION } ?

Comment (etiennehomer) [reply src/main/java/org/gridsuite/modification/server/dto/TabularModificationInfos.java:45]: You want to use 2 fields : IdentifiableType equipmentType +  OperationType operationType ?

Comment (dbraquart) [reply src/main/java/org/gridsuite/modification/server/dto/TabularModificationInfos.java]: Last decision: we use ModificationType enum. No data migration. Same Front-end.

**Review by SlimaneAmar — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]
Tests OK

**Pull request merged**